### ISSUE TITLE
Unsupported Ops for tf.layers.dense and tf.layers.dropout

### ISSUE THREAD

**Issue by georgwiese** [tensorflow/tfjs#168]
I'm trying to export an existing TF `SavedModel` using the `tensorflowjs_converter` tool. The conversion fails with the message:

```
Unsupported Ops in the model
Merge, Prod, Switch, ListDiff
```

As far as I can tell, `Switch` and `Merge` are created by `tf.layers.dropout()`. The other two come from `tf.layers.dense()` (the Ops are in the `Tensordot` scope).

Both of these seem like standard operations needed in most networks, so I'm wondering if I'm overlooking something. Is there a workaround I'm not aware of?

**Comment by georgwiese** [comment]
Looks like `tf.layers.dense()` only reproduces the `Prod` and `ListDiff` ops [when used with input tensors of rank bigger than 3](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/layers/core.py#L156) (as part of `tensorflow.python.ops.tensordot`).

Here's a minimal example to reproduce:

```python
import tensorflow as tf

sess = tf.Session()
x = tf.placeholder(tf.float32, shape=[None, 3, 3])
y = tf.layers.dense(x, 1)

sess.run(tf.global_variables_initializer())
print(y.name)

tf.saved_model.simple_save(sess, "saved_model", {"x": x}, {"y": y})
```

Then in command line:
```bash
$ tensorflowjs_converter \
   --input_format tf_saved_model \
   --output_node_names "dense/BiasAdd" \
   saved_model_reproduce foo
```

Output:
```
Converted 2 variables to const ops.
Unsupported Ops in the model
ListDiff, Prod
```

**Comment by nsthorat** [comment]
@pyu10055 can you revisit which of these are done?
